src/ui/pages/CredentialDetails/CredentialDetails.tsx

```diff
@@ -96,7 +96,7 @@ const CredentialDetails = () => {
       const creds = await Agent.agent.credentials.getCredentials(true);
       dispatch(setCredsArchivedCache(creds));
     } catch (e) {
-      // @TODO - duke: handle error
+      dispatch(setToastMsg(ToastMsgType.GET_ARCH_CRED_FAIL));
```

Comment:
There should be no toast message here. This can be handled in the console and just catch the error.

The one I was asking for is when you want to archive a credential `handleArchiveCredential`